app/models/state_file_base_intake.rb

```diff
@@ -1,5 +1,5 @@
 class StateFileBaseIntake < ApplicationRecord
-  self.ignored_columns = [:df_data_import_failed]
+  self.ignored_columns = [:df_data_import_failed, :bank_name]
```

Comment:
if we add bank_name to the shared base intake then we don't need to add it for Az right?

Reply:
^ i think so too

Reply:
unfortunately, the `ignored_columns` get overwritten by the AZIntakes version. I can refactor to use the existing ignored columns and _add_ the other attributes being ignored in AZ

Reply:
refactor: https://github.com/codeforamerica/vita-min/pull/5007/commits/b2219bd9849498259336b95b2a598e0680be77c0